### ISSUE TITLE
AttributeError: type object 'str' has no attribute 'str', when printing UTC_time variable of https://www.unidata.ucar.edu/software/netcdf/examples/test_hgroups.nc

### ISSUE THREAD

**Issue by jjrob** [Unidata/netcdf4-python#439]
[Unidata's list of example netCDF-4 files](https://www.unidata.ucar.edu/software/netcdf/examples/files.html) includes [test_hgroups.nc](https://www.unidata.ucar.edu/software/netcdf/examples/test_hgroups.nc). netCDF4-python raises an AttributeError when trying to print the UTC_time variable from this file:

```python
>>> import sys
>>> sys.version
'2.7.8 (default, Jun 30 2014, 16:03:49) [MSC v.1500 32 bit (Intel)]'
>>> import netCDF4
>>> netCDF4.__version__
'1.0.8'
>>> root = netCDF4.Dataset(r'C:\Temp\test_hgroups.nc')
>>> root.variables
OrderedDict([(u'UTC_time', <netCDF4.Variable object at 0x071D1260>)])
>>> print root.variables['UTC_time']
Traceback (most recent call last):
  File "<interactive input>", line 1, in <module>
  File "netCDF4.pyx", line 2451, in netCDF4.Variable.__str__ (netCDF4.c:30399)
  File "netCDF4.pyx", line 2489, in netCDF4.Variable.__unicode__ (netCDF4.c:31167)
AttributeError: type object 'str' has no attribute 'str'
```

This is the win32 build of netCDF4-python that comes with ArcGIS 10.3.1, the latest version of ArcGIS. Starting with ArcGIS 10.3, ESRI builds and packages netCDF4-python with their copy of Python 2.7. I have not tried this with a newer build of netCDF4-python. Sorry for not doing that work; I'm hoping that a developer can quickly repro this or confirm it does not repro with a more recent build...

This is not a critical problem for me but I'm reporting it so you know about it. Thanks for your time.

**Comment by shoyer** [comment]
I cannot reproduce this using netCDF4-python 1.1.5 on OS X.

**Comment by jjrob** [comment]
Thank you for your quick check. I'm closing it under the assumption that it was fixed sometime between 1.0.8 and 1.1.5.
